libstuff/sqlite3ext.h

```diff
@@ -319,6 +319,50 @@ struct sqlite3_api_routines {
                             void(*xDestroy)(void*));
   /* Version 3.26.0 and later */
   const char *(*normalized_sql)(sqlite3_stmt*);
+  /* Version 3.28.0 and later */
```

Comment:
@tylerkaraszewski In the [update SQLite instructions](https://stackoverflowteams.com/c/expensify/questions/154) this file `sqlite3ext.h` is not mentioned, but I added it because there were changes. Do you know if this is fine or if I should remove it from the PR?